lib/storage/acl.js

```diff
@@ -232,28 +233,30 @@ nodeutil.inherits(Acl, AclRoleAccessorMethods);
  * }, function(err, aclObject, apiResponse) {});
  */
 Acl.prototype.add = function(options, callback) {
-  var that = this;
-
-  var body = {
-    entity: options.entity,
-    role: options.role.toUpperCase()
-  };
-
-  var query = null;
+  var self = this;
 
+  var query = {};
   if (options.generation) {
     query = {
       generation: options.generation
```

Comment:
kind of a nitpick here, if we're now setting query as an empty object, wouldn't it be better to just assign the generation property rather than completely blow away the previous object?